tls/s2n_connection.c

```diff
@@ -554,10 +467,21 @@ int s2n_connection_wipe(struct s2n_connection *conn)
     }
     POSIX_GUARD_RESULT(s2n_prf_wipe(conn));
     struct s2n_prf_working_space *prf_workspace = conn->prf_space;
+    if (!conn->initial) {
```

Comment:
Moving the initialization of the crypto params to s2n_connection_wipe means that there would be more frequent calls to allocate memory right? Do we need documentation for this kind of thing?

Reply:
Only if s2n_connection_free_handshake is called. Otherwise, we just reuse the memory allocated by s2n_connection_new.